commcare_export/cli.py

```diff
@@ -155,6 +155,21 @@ def _get_writer(output_format, output, strict_types):
         raise Exception("Unknown output format: {}".format(output_format))
 
 
+def get_since_until(args, checkpoint_manager):
+    if not args.since and not args.start_over and os.path.exists(args.query) and checkpoint_manager:
+        with checkpoint_manager:
+            args.since = checkpoint_manager.get_time_of_last_run()
+
+        if args.since:
+            logger.debug('Last successful run was %s', args.since)
+        else:
+            logger.warn('No successful runs found, and --since not specified: will import ALL data')
+
+    since = dateutil.parser.parse(args.since) if args.since else None
```

Comment:
Same comment as above! I think a more descriptive var name name for `since` and `until` would help me follow this